### PR TITLE
Sample: serial_lte_modem AT#XSLEEP update rollback

### PR DESCRIPTION
**Pull request by junqingzou**
Rollback AT#XSLEEP update in [PR#1732](https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1732) PR#1732 due to application-only sleep
was tested to have affected modem cellular connection negatively.

Back to simple sleep of whole nRF9160, while application idle mode
is to be implemented later.

Add GPIO wakeup to all UART instances, support wakeup by PCA10090
Button_1 when UART_0 is used for AT channel.

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

File: samples/nrf9160/serial_lte_modem/Kconfig [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-source "$ZEPHYR_BASE/Kconfig.zephyr"
```
Comment (rlubos) [inline samples/nrf9160/serial_lte_modem/Kconfig]: Is this change intended? The removed version looks fine?

File: samples/nrf9160/serial_lte_modem/README.rst [hunk 2]
```diff
@@ -95,7 +95,7 @@ Generic commands
 The following proprietary generic AT commands are used in this sample:
 
 * AT#XSLMVER
-* AT#XSLEEP=<mode>
+* AT#XSLEEP
```
Comment (ru-fu) [inline samples/nrf9160/serial_lte_modem/README.rst]: This was "AT#XSLEEP (only for external MCU configuration)" before the change.
Not sure if the extra information is needed?

File: samples/nrf9160/serial_lte_modem/Kconfig (hunk 1 above)
Comment (junqingzou) [reply samples/nrf9160/serial_lte_modem/Kconfig]: Yes, Nits issues before. "Redundant 'source "$(ZEPHYR_BASE)/Kconfig.zephyr" in 'samples/nrf9160/serial_lte_modem/Kconfig'. Just do 'source "Kconfig.zephyr"'
instead. The $srctree environment variable already points to the Zephyr root,
and all 'source's are relative to it."

File: samples/nrf9160/serial_lte_modem/README.rst (hunk 2 above)
Comment (junqingzou) [reply samples/nrf9160/serial_lte_modem/README.rst]: Based on customer feedback, Sleep/Wakeup is enabled for PC client as well. So this comment does not stand any more.

Comment (ru-fu) [reply samples/nrf9160/serial_lte_modem/README.rst]: ok
